Doc/c-api/veryhigh.rst

```diff
@@ -183,7 +183,7 @@ the same library that the Python runtime is using.
    objects *globals* and *locals* with the compiler flags specified by
    *flags*.  *globals* must be a dictionary; *locals* can be any object
    that implements the mapping protocol.  The parameter *start* specifies
-   the start token that should be used to parse the source code.
+   the start symbol that should be used to parse the source code.
```

Comment:
Let's also provide a link to [`Py_eval_input`](https://docs.python.org/3/c-api/veryhigh.html#c.Py_eval_input), so the user does not have to search through this page in order to find out _what_ a "start symbol" is :)